integration-scripts/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@chainlink/integration-scripts",
+  "version": "0.0.1",
+  "description": "Scripts for helping perform integration tests",
+  "main": "",
+  "repository": "https://github.com/smartcontractkit/chainlink",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "generate-typings": "typechain --target ethers --outDir src/generated \"build/contracts/*\"",
+    "build:contracts": "truffle compile",
+    "postbuild:contracts": "yarn generate-typings",
+    "prebuild": "yarn build:contracts",
+    "build": "tsc",
+    "setup": "yarn build",
+    "format": "prettier --write \"src/**/*\"",
+    "lint": "eslint --ext .js,.ts \"src/**/*\"",
+    "count-transaction-events": "node ./dist/countTransactionEvents",
+    "send-runlog-transaction": "node ./dist/sendRunlogTransaction",
+    "send-ethlog-transaction": "node ./dist/sendEthlogTransaction",
+    "fund-address": "node ./dist/fundAddress",
+    "deploy-contracts": "node ./dist/deployContracts",
+    "start-echo-server": "node ./dist/echoServer"
+  },
+  "devDependencies": {
+    "@chainlink/eslint-config": "0.0.1",
+    "@chainlink/prettier-config": "^0.0.1",
+    "@types/body-parser": "^1.17.1",
+    "@types/express": "^4.17.1",
+    "eslint": "6.3.0",
+    "typechain": "1.0.0-beta.2",
+    "typechain-target-ethers": "^1.0.0-beta.1",
+    "typescript": "^3.6.3"
+  },
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "link_token": "^1.0.6",
+    "ethers": "^4.0.37",
+    "express": "^4.16.2",
```

Comment:
The explorer requires `^4.16.4`. How do you feel about making this the same?

Reply:
Works for me